ncm-metaconfig/src/main/metaconfig/opennebula/tests/regexps/config/base

```diff
@@ -0,0 +1,10 @@
+Base test for config
+---
+multiline
+/etc/aii/opennebula.conf
+---
+^\[.+\]$
+^password\=.+$
+^pattern\=.+$
```

Comment:
you can actually make a test with the testvalues included, to be sure everything renders correctly (with/without quotes, ints, booleans)